lib/src/structs.dart

```diff
@@ -3070,106 +3364,408 @@ class ACTCTX extends Struct {
+  @Uint64()
+  external int _wchar196_szInfo;
+  @Uint64()
+  external int _wchar200_szInfo;
+  @Uint64()
+  external int _wchar204_szInfo;
+  @Uint64()
+  external int _wchar208_szInfo;
+  @Uint64()
+  external int _wchar212_szInfo;
+  @Uint64()
+  external int _wchar216_szInfo;
+  @Uint64()
+  external int _wchar220_szInfo;
+  @Uint64()
+  external int _wchar224_szInfo;
+  @Uint64()
+  external int _wchar228_szInfo;
+  @Uint64()
+  external int _wchar232_szInfo;
+  @Uint64()
+  external int _wchar236_szInfo;
+  @Uint64()
+  external int _wchar240_szInfo;
+  @Uint64()
+  external int _wchar244_szInfo;
+  @Uint64()
+  external int _wchar248_szInfo;
+  @Uint64()
+  external int _wchar252_szInfo;
+
+  @Uint32()
+  external int uTimeout;
+
+  @Uint32()
+  external int uVersion; // used with NIM_SETVERSION, values 0, 3 and 4
+
+  // WCHAR szInfoTitle[64]
+  @Uint64()
+  external int _wchar0_szInfoTitle;
+  @Uint64()
+  external int _wchar4_szInfoTitle;
+  @Uint64()
+  external int _wchar8_szInfoTitle;
+  @Uint64()
+  external int _wchar12_szInfoTitle;
+  @Uint64()
+  external int _wchar16_szInfoTitle;
+  @Uint64()
+  external int _wchar20_szInfoTitle;
+  @Uint64()
+  external int _wchar24_szInfoTitle;
+  @Uint64()
+  external int _wchar28_szInfoTitle;
+  @Uint64()
+  external int _wchar32_szInfoTitle;
+  @Uint64()
+  external int _wchar36_szInfoTitle;
+  @Uint64()
+  external int _wchar40_szInfoTitle;
+  @Uint64()
+  external int _wchar44_szInfoTitle;
+  @Uint64()
+  external int _wchar48_szInfoTitle;
+  @Uint64()
+  external int _wchar52_szInfoTitle;
+  @Uint64()
+  external int _wchar56_szInfoTitle;
+  @Uint64()
+  external int _wchar60_szInfoTitle;
+
+  @Uint32()
+  external int dwInfoFlags;
+
+  @Uint32()
+  external int guidItem1;
+  @Uint16()
+  external int guidItem2;
+  @Uint16()
+  external int guidItem3;
+  @Uint64()
+  external int guidItem4;
+
+  @IntPtr()
+  external int hBalloonIcon;
+}
+
+extension Pointer_NOTIFYICONDATA_Extension on Pointer<NOTIFYICONDATA> {
+  String get szTip =>
+      cast<Uint8>().elementAt(40).cast<Utf16>().unpackString(128);
+
+  set szTip(String text63chars) {
+    if (text63chars.length > 63) {
+      throw ArgumentError('Argument "text64chars" should be less 63 char');
+    }
+    _wchar_cpy(this, 40, text63chars);
+  }
+
+  String get szInfo =>
+      cast<Uint8>().elementAt(304).cast<Utf16>().unpackString(256);
+
+  set szInfo(String text255chars) {
+    if (text255chars.length > 255) {
+      throw ArgumentError('Argument "text255chars" should be less 255 char');
+    }
+    _wchar_cpy(this, 304, text255chars);
+  }
+
+  String get szInfoTitle =>
+      cast<Uint8>().elementAt(824).cast<Utf16>().unpackString(64);
+
+  set szInfoTitle(String text63chars) {
+    if (text63chars.length > 63) {
+      throw ArgumentError('Argument "text63chars" should be less 63 char');
+    }
+    _wchar_cpy(this, 824, text63chars);
+  }
+
+  static void _wchar_cpy(Pointer pointer, int memStart, String source) {
```

Comment:
This is a neat approach. But there's a similar one in extensions/set_string.dart, and I wonder if we should merge them together. We won't want a static method like this in every extension method that sets strings, I don't think.